src/vs/workbench/services/editor/common/editorGroupsService.ts

```diff
@@ -477,7 +482,7 @@ export interface IEditorGroup {
 	 * @returns a promise that resolves around an IEditor instance unless
 	 * the call failed, or the editor was not opened as active editor.
 	 */
-	openEditor(editor: IEditorInput, options?: IEditorOptions | ITextEditorOptions, context?: OpenEditorContext): Promise<IEditorPane | null>;
+	openEditor(editor: IEditorInput, options?: IEditorOptions | ITextEditorOptions, openContext?: OpenEditorContext): Promise<IEditorPane | undefined>;
```

Comment:
I feel that `openContext` should not be exposed as API and that the new event should fire from within the group when we know a move or copy happens.

Reply:
This should go away if we move the event.

Reply:
Gone